README.md

```diff
@@ -14,88 +26,150 @@ To get started with the Ada Trader project follow these steps:
-
-1. The `QuoteView` class should:
-  * Setup [an event listener](http://backbonejs.org/#Events-on) for a custom event named `change:price`. This should be done in the `initialize` function.
-  * Have a function that runs when the `change:price` event happens. This function should:
-    * Expect that a `change` parameter will be passed into it.
-    * Expect that the `change` parameter will be a number, positive OR negative.
-    * Add the `change` parameter onto the current stock price.
-    * Re-render the view so that the new price is displayed to the user.
-1. Something in the project should call the `simulate` function on each of the `QuoteView` instances.
-
-You may find that you would rather move the `simulate` function out of `app.js`. Feel free to do so.
-
-We do not expect you to write unit tests for the `simulate` function. -->
+### Project Baseline
+This project uses the same structure as BackTREK and is based on our [backbone baseline](https://github.com/AdaGold/backbone-baseline). The baseline for this project includes some model and collections that have already been implemented:
+
+| Model | File | Purpose |
+|:------|:-----|:--------|
+| `Quote` | `src/models/quote.js` | This model represents an individual stock quote, and has attributes for:<br>• `symbol` - code name for the stock (e.g. AAPL)<br>• `price` - current price of the quote<br>You will add the ability to buy / sell a quote using custom functions. |
+| `QuoteList` | `src/collections/quote_list.js` | This collection represents all of the quotes available on the trading platform. |
+| `Simulator` | `src/models/simulator.js` | This model handles the logic for simulating stock market activity by randomly shifting the price for each quote up or down each second. You will need to connect it to your `QuoteList` instance in `src/app.js`. |
+
+# Project Requirements
+## Wave 1 - Trading with Market Orders
+### Trading Overview
+In this wave you will implement the Backbone Views, events, and model logic to allow a user to trade stocks using "[market orders](#trading-vocabulary)". A market order is when a trader purchases or sells a stock at the _current_ price, and the order "executes" (actually becomes a trade) immediately.
+
+### Interface Details
+In the Ada Trader interface market orders are created by clicking the Buy and Sell buttons next to each stock quote. When a user presses one of these buttons they are trading that stock at the price displayed at that moment.
+
+### User Stories
+As a user, I can:
+  - See a list of quotes in the left side of the top panel, including the:
+    - Symbol for each quote
+    - [Market price](#trading-vocabulary) for each quote
+    - Buy button for each quote
+    - Sell button for each quote
+  - Click the Buy button associated with a quote
+  - Click the Sell button associated with a quote
+As a user, when I:
+  - Click the Buy button for a quote:
+    - That quote's market price increases by $1.00
```

Comment:
The question that I have off the bat is how does this price change impact/interact with the prices coming from the simulator?